Gopkg.toml

```diff
@@ -21,15 +21,15 @@ required = [
   name = "knative.dev/pkg"
   branch = "master"
 
+# This is a preemptive override.
```

Comment:
What does this comment mean? Why is knative.dev/pkg not commented with this? I just don't understand what preemptive override mean.